### PR TITLE
recursive-for

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: staff/juan-sensio/safe-box/safe-box.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+})();
+
+/**
+ * Check if the password is a correct string
+ * 
+ * @param {string} password The password
+ * 
+ * @throws {Error} If password is wrong
+ */
+function checkPassword(password, msg) {
```
Comment (manuelbarzi) [inline staff/juan-sensio/safe-box/safe-box.js:62]: i would rather hide this check-password function inside the 'selfie', so it doesn't pollute the global scope ,)

File: staff/juan-sensio/spotify-app/scripts/logic.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+var logic = {
```
Comment (kimbali) [inline staff/juan-sensio/spotify-app/scripts/logic.js:1]: 'use strict'

Comment (ghost) [reply staff/juan-sensio/spotify-app/scripts/logic.js:1]: @kimbali done !

File: staff/juan-sensio/workers/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Workers
```
Comment (jaumeserr) [inline staff/juan-sensio/workers/README.md:1]: beautiful README.md

Comment (ghost) [reply staff/juan-sensio/workers/README.md:1]: @jaumeserr ty

File: staff/juan-sensio/spotify-app/scripts/logic.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+    retrieveTracksByAlbumId(id) {
+        return this._callApi('/albums/' + id + '/tracks')
+            .then(function(res) {
+                return res.items;
+            });
+    },
+
+    retrieveTrackById(id) {
+        // TODO
```
Comment (jaumeserr) [inline staff/juan-sensio/spotify-app/scripts/logic.js:52]: Where is the code?

Comment (ghost) [reply staff/juan-sensio/spotify-app/scripts/logic.js:52]: @jaumeserr 🤫

**Pull request merged**